### ISSUE TITLE
Missing metadata.creationTimestamp

### ISSUE THREAD

**Issue by urbaniak** [clux/kube-rs#93]
Need to access creationTimestamp of pods metadata and figured out it's missing from the `ObjectMeta` struct.

Is it possible to add it?

**Comment by clux** [comment]
yeah, of course! feel free to submit a pr with the 1/2 lines and i'll merge. Otherwise i'm sure there'll be a release some time this / next week.

**Comment by clux** [comment]
Ok, had a quick go at this.

It would either introduce the dependency on:

```
use k8s_openapi::apimachinery::pkg::apis::meta::v1::{Time};
```

to decode the timestamp correctly, or we inline that same code :/

But we already use that code in the non-default feature `openapi`. Might just release it under the openapi feature only for now.

**Comment by clux** [comment]
For now, have hidden it behind `openapi`, but might inline https://docs.rs/k8s-openapi/0.6.0/src/k8s_openapi/v1_16/apimachinery/pkg/apis/meta/v1/time.rs.html#5 in this crate in the future. We've copied the generic parts from there before.

**Comment by clux** [comment]
Released in 0.22.0

**Comment by clux** [comment]
> For now, have hidden it behind `openapi`, but might inline https://docs.rs/k8s-openapi/0.6.0/src/k8s_openapi/v1_16/apimachinery/pkg/apis/meta/v1/time.rs.html#5 in this crate in the future. We've copied the generic parts from there before.
